examples/org-viewer-react/src/environment.ts

```diff
@@ -22,8 +21,6 @@ const ENVIRONMENTS = new Map([
     'mainnet_staging',
```

Comment:
Should we remove the staging environment as we did for the `org-viewer-web` example?